projects/plugins/jetpack/tests/e2e/lib/pages/wp-page.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Internal dependencies
+ */
+import PageActions from './page-actions';
+
+export default class WpPage extends PageActions {
```

Comment:
I am not totally against this approach, but I'd like to note that inheritance might not be the best fit here. Since `PageActions` is basically a wrapper around `pw` APIs and its nothing to do with the representation of the page, using composition could be more natural. 

Although, I see a benefit of using inheritance approach here - the calls to wrapper methods looks shorter: `this.waitForDomContentLoaded()` vs `this.actions.waitForDomContentLoaded()`

Reply:
I see what you mean. For me though, it feels natural to call all these actions on `this`, `this` being a page representation. I tried to emulate the PW api as close as possible, where you call these methods on page: `page.click()`.
But, same as you, I'm not totally against composition either.